syscall/devices/hostepoll/CMakeLists.txt

```diff
@@ -5,8 +5,6 @@ add_enclave_library(oehostepoll STATIC hostepoll.c)
 
 maybe_build_using_clangw(oehostepoll)
 
-add_enclave_dependencies(oehostepoll syscall_trusted_edl)
```

Comment:
For my understanding, these can all be removed because they were always getting this dependency transitively through the link dependency on oesyscall anyway, correct?

Reply:
correct